test/cache.test.js

```diff
@@ -156,41 +156,6 @@ test.serial.cb(
   },
 );
 
-test.serial.cb(
-  "should output files to standard cache dir if set to true in query",
```

Comment:
This is a test on cache behaviour. Can you adapt the query string to options so we don't have to remove this test?

Reply:
Whoops, sorry about that! Fixed in [b9cc145](https://github.com/babel/babel-loader/pull/942/commits/b9cc14551d9fc3e6b317af8644a6c112bdd37b36).